doc/nrf/releases_and_maturity/migration/migration_guide_2.6.rst

```diff
@@ -40,6 +40,25 @@ The following changes are mandatory to make your application work in the same wa
     You can also have the DAC private key replaced by zeros in the factory data partition by setting the :kconfig:option:`CONFIG_CHIP_CRYPTO_PSA_MIGRATE_DAC_PRIV_KEY` Kconfig option to ``y``.
     This functionality is experimental.
 
+* For the Matter samples and applications using :file:`samples/matter/common` directory:
+
+  * The structure of the files located in the :file:`common` directory has been changed.
+    You should align the appropriate paths in your application's :file:`CMakeLists.txt` file and application's source files including header files located in the :file:`common` directory.
+  * The :file:`event_types.h` file was removed.
+    If your application uses it, you should add the :file:`event_types.h` file in your application :file:`src` directory, that contains the following code:
+
+    .. code-block:: C++
+
+      #pragma once
+
+      struct AppEvent; /* needs to be implemented in the application code */
+      using EventHandler = void (*)(const AppEvent &);
```

Comment:
```suggestion
       #pragma once

       struct AppEvent; /* needs to be implemented in the application code */
       using EventHandler = void (*)(const AppEvent &);
```
Don't think the one-space difference affects the build, but might as well get it just right.